docs/agents/sematext-agent/agent-troubleshooting.md

```diff
@@ -0,0 +1,38 @@
+title: Agent Troubleshooting
+
+If you are experiencing issues with the Sematext Agent, there are some things you can try.
+
+1. Check the Sematext Agent logs and filter by errors. By default they'll be located at `/opt/spm/spm-monitor/logs/st-agent`. 
+   1. If you see errors related to permission denied while accessing log files or system information, [stop the agent and start it](https://sematext.com/docs/agents/sematext-agent/starting-stopping) again with [elevated privileges](https://sematext.com/docs/agents/sematext-agent/permission-requirements/). 
+   2. If you see errors like `"cloud" collector failed to complete the initial run` check out our [tags common schema docs](https://sematext.com/docs/tags/common-schema/#cloud-tags) and verify your IAM roles are set up correctly.
```

Comment:
We are referring to IAM, but IAM is AWS specific and cloud tags are collected for GCE and Azure, too.

Reply:
IAM stands for Identity and access management, it's the same in [GCP](https://cloud.google.com/iam/docs/understanding-roles), [Azure](https://docs.microsoft.com/en-us/azure/role-based-access-control/role-assignments-portal?tabs=current) and other prividers